### PR TITLE
Add ability to gets all active connections to the database.

### PR DESCRIPTION
**Pull request by djgoku**
This adds the ability to get all active connections to the database.

Screenshot 2022-11-17 at 09 17 28

### CODE REVIEW

**Comment by djgoku** [comment]
I'll be pushing an update to this tomorrow morning and adding a screenshot too.

**Comment by djgoku** [comment]
I am done pushing changes to this PR.

File: test/ecto_psql_extras_test.exs [hunk 1]
```diff
@@ -13,7 +13,7 @@ defmodule EctoPSQLExtrasTest do
     outliers_legacy: EctoPSQLExtras.OutliersLegacy
   }
 
-  @skip_queries [:kill_all]
+  @skip_queries [:kill_all, :connections]
```
Comment (pawurb) [inline test/ecto_psql_extras_test.exs]: Why do you skip it in the test?

Comment (djgoku) [reply test/ecto_psql_extras_test.exs]: This used to be just a count. Looking into adding a test now.

Comment (djgoku) [reply test/ecto_psql_extras_test.exs]: I removed it in the latest push.

**Review by pawurb — APPROVED** [review]

**Comment by siphonhata** [comment]
Good Day, since I'm able to see the active connections. Is it possible to closed a connection whether its busy or not

**Pull request merged**